controller/test-files/area/show/if_modified_since_headers.res.headers.golden.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Cache-Control": [
+    "private,max-age=120"
+  ],
+  "Etag": [
+    "IGNORE_ME"
```

Comment:
Can you please use `safeOverriteHeader(t, res, app.ETag, "0icd7ov5CqwDXN6Fx9z18g==")` to set the `app.Etag` header? It came in not so long ago and should simplify setting the Etag in a safe manner, e.g. if it didn't exist before. Also, by doing so we have a list an Etag that looks like a proper Etag ;)

Reply:
Please see https://github.com/fabric8-services/fabric8-wit/pull/1941/commits/e46548e2d6bea7bac73911a818b127df8643e240